AMM/contracts/AMM/tests/functions/add_pool.rs

```diff
@@ -0,0 +1,119 @@
+use crate::utils::{
+    amm_abi_calls::{add_pool, pool},
+    test_helpers::{deploy_and_construct_exchange_contract, setup, setup_and_initialize},
+};
+
+mod success {
+    use super::*;
+
+    #[tokio::test]
+    async fn adds() {
+        let (wallet, amm_instance, assets) = setup_and_initialize().await;
+        let pair = (assets[0], assets[1]);
```

Comment:
Does it make sense to change `assets` to contain two different pairs so that setting up in each test is a little shorter?

Reply:
Sure. Done.